src/domain/event/eventInfo/EventInfo.tsx

```diff
@@ -52,39 +60,60 @@ const EventInfo: React.FC<Props> = ({ event, eventType }) => {
   });
 
   const {
-    addressLocality,
-    district,
     email,
-    endTime,
     externalLinks,
-    googleDirectionsLink,
-    hslDirectionsLink,
     infoUrl,
     languages,
-    locationName,
-    name,
-    offerInfoUrl,
-    shortDescription,
-    startTime,
-    streetAddress,
     telephone,
     audience,
-    registrationUrl,
   } = getEventFields(event, locale);
-
-  const eventPriceText = getEventPrice(
-    event,
-    locale,
-    t('event.info.offers.isFree')
-  );
-
+  
+  const { audienceMinAge, audienceMaxAge } = event;
+  
   const showOtherInfo = Boolean(
     email || externalLinks.length || infoUrl || telephone
   );
 
-  const moveToBuyTicketsPage = () => {
-    window.open(offerInfoUrl);
-  };
+  return (
+    <div className={styles.eventInfo} ref={eventInfoContainer}>
+      <div className={styles.contentWrapper}>
+        <DateInfo event={event} eventType={eventType} />
+        <SuperEvent superEvent={superEvent} />
+        <SubEvents event={event} />
+        <OtherEventTimes event={event} />
+        {(audienceMinAge || audienceMaxAge) && <AudienceAgeLimitations audienceMinAge={audienceMinAge} audienceMaxAge={audienceMaxAge} eventType={eventType} />}
+        <LocationInfo event={event}/>
```

Comment:
Prettie not used?

Reply:
I had some unexpected problems with editor's auto formatter and it is committed with no-verify. Need to run prettier everywhere when committing tests.

Reply:
Run prettier to all files. Should be ok now.